src/v2/components/ReviewerDashboard/ApplicationsTable.tsx

```diff
@@ -0,0 +1,218 @@
+import React, { useContext, useEffect, useState } from 'react'
+import {
+	Box,
+	Button,
+	Center,
+	Flex,
+	Grid,
+	GridItem,
+	Table,
+	Tbody,
+	Td,
+	Text,
+	Th,
+	Thead,
+	Tr,
+} from '@chakra-ui/react'
+import { useRouter } from 'next/router'
+import { ApiClientsContext } from '../../../../pages/_app'
+import Loader from '../../../components/ui/loader'
+import { defaultChainId } from '../../../constants/chains'
+import { ApplicationState, useGetReviewerApplicationsQuery } from '../../../generated/graphql'
+import { getFormattedDateFromUnixTimestampWithYear } from '../../../utils/formattingUtils'
+import { capitalizeFirstLetter } from '../../../utils/generics'
+import { getSupportedChainIdFromWorkspace } from '../../../utils/validationUtils'
+import PaginatorView from '../WorkspaceMembers/PaginatorView'
+
+const PAGE_SIZE = 5
+
+type Application = {
+  id: string,
+  submittedOn: number,
+  state: ApplicationState,
+  projectName: string,
+  applicantId: string,
+  applicantName: string,
+  applicantEmail?: string,
+}
+
+type Props = {
+  reviewerId: string,
+  header: string,
+  showApplicationState: boolean,
+  showReviewButton: boolean,
+  applicationStateIn?: ApplicationState[],
```

Comment:
does it need to be optional?

Reply:
if not passed, it'll take all enum values..